### PR TITLE
PR for unFederal Reserve

### PR DESCRIPTION
**Pull request by CryptoBiased**
Pull Request for unFederal Reserve

Add space for unFederal Reserve

Etherscan: https://etherscan.io/address/0x5218e472cfcfe0b64a064f055b43b4cdc9efd3a6
CoinGecko: Coin Gecko https://www.coingecko.com/en/coins/unfederalreserve
Website: https://app.unfederalreserve.com
Twitter: https://twitter.com/Unfederalreser1

### CODE REVIEW

File: skins/unfederalreserve.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  --text-color: #002346;
+  --link-color: #004a94;
+  --heading-color: #28263b;
+  --block-bg: transparent;
+
+  #topnav {
+    border-bottom: 4px solid black !important;
+    border-image-slice: 1;
+    border-image-source: linear-gradient(to left, #004a94, #587bdf);
+  }
```
Comment (gokaiorg) [inline skins/unfederalreserve.scss:15]: Please use only theme variables. No extra custom styles are allowed.

File: spaces/iyffinance/index.json [hunk 2]
```diff
@@ -1,20 +1,13 @@
...
-    "defaultTab": "core",
-    "minScore": 10
-  }
+  "defaultView": "core",
+  "token": "0x5d762f76b9e91f71cc4f94391bdfe6333db8519c",
+  "core": [],
+  "min": 10,
+  "invalid": [],
+  "skin": "iyffinance"
 }
```
Comment (gokaiorg) [inline spaces/iyffinance/index.json:13]: You have added a file from another space, please delete it.

File: spaces/unfederalreserve/index.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "unFederal Reserve",
+  "chainId": 1,
+  "decimals": 18,
+  "symbol": "eRSDL",
+  "defaultView": "all",
+  "token": "0x5218e472cfcfe0b64a064f055b43b4cdc9efd3a6",
```
Comment (gokaiorg) [inline spaces/unfederalreserve/index.json:7]: You will have to remove the token field. We do not use it anymore.

File: spaces/unfederalreserve/index.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  "core": [
+    "0xF4E89104371026FA7DDA07Ef7c839A44f95efC90",
+    "0x5fd9C4368ee7E8fc79e48fBa390c148fd16453Ee"
+  ],
+  "min": 100000,
+  "invalid": [],
+  "skin": "unfederalreserve"
+}
```
Comment (gokaiorg) [inline spaces/unfederalreserve/index.json:15]: You will miss the strategies field. Please refer to the doc to complete your space https://docs.snapshot.page/guides/create-a-space#3-space-metadata

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/unfederalreserve/index.json (hunk 3 above)
Comment (CryptoBiased) [reply spaces/unfederalreserve/index.json:7]: ok... so this documentation is where I saw that "Token" was required:  https://docs.snapshot.page/guides/create-a-space#required-fields-token-name-network-symbol-strategies

File: spaces/iyffinance/index.json (hunk 2 above)
Comment (CryptoBiased) [reply spaces/iyffinance/index.json:13]: Not quite sure how that happened, but will address!

File: skins/unfederalreserve.scss (hunk 1 above)
Comment (CryptoBiased) [reply skins/unfederalreserve.scss]: Done.

File: spaces/unfederalreserve/index.json (hunk 3 above)
Comment (CryptoBiased) [reply spaces/unfederalreserve/index.json]: Done.

File: spaces/unfederalreserve/index.json (hunk 4 above)
Comment (CryptoBiased) [reply spaces/unfederalreserve/index.json]: Added strategies.  Done.

**Comment by CryptoBiased** [comment]
Continuity errors.  Will re-issue PR from new Branch.

**Pull request closed without merging**